azure_functions_worker/bindings/meta.py

```diff
@@ -25,16 +29,34 @@ def load_binding_registry() -> None:
     global BINDING_REGISTRY
     BINDING_REGISTRY = func.get_binding_registry()
 
+    # The base extension supports python 3.8+
+    if sys.version_info.minor >= BASE_EXT_SUPPORTED_PY_MINOR_VERSION:
+        # Import the base extension
+        try:
+            import azure.functions.extension.base as clients
+            global DEFERRED_BINDINGS_REGISTRY
+            DEFERRED_BINDINGS_REGISTRY = clients.get_binding_registry()
+        except ImportError:
+            # This means that the customer hasn't imported the library.
+            # This isn't an error.
+            pass
 
-def get_binding(bind_name: str) -> object:
-    binding = None
-    registry = BINDING_REGISTRY
-    if registry is not None:
-        binding = registry.get(bind_name)
-    if binding is None:
-        binding = generic.GenericBinding
 
-    return binding
+def get_binding(bind_name: str, pytype: typing.Optional[type] = None) -> object:
+    try:
+        # Check if binding is deferred binding
+        binding = get_deferred_binding(bind_name=bind_name, pytype=pytype)
+        # Binding is not deferred binding type
+        if binding is None:
+            binding = BINDING_REGISTRY.get(bind_name)
+        # Binding is generic
+        if binding is None:
+            binding = generic.GenericBinding
+        return binding
+    except AttributeError:
```

Comment:
Add this check in init request

Reply:
Moved to load_binding_registry, which is called in init and env reload